webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+function SimpleContentAccess(props) {
+  const {
+    canToggleSimpleContentAccess,
+    simpleContentAccessValue,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    setActionInProgress,
+  } = props;
+
+  const [isChecked, setChecked] = useState(simpleContentAccessValue);
+
+  useEffect(() => {
+    setChecked(simpleContentAccessValue);
+  });
+
+  function toggleSimpleContentAccess() {
+    if (simpleContentAccessValue) {
+      disableSimpleContentAccess();
+    } else {
+      enableSimpleContentAccess();
+    }
+  }
+
+  function handleChange() {
+    setChecked(!isChecked);
+    setActionInProgress();
+    toggleSimpleContentAccess();
+    props.hideModal();
```

Comment:
This happens too fast and it's a bit jarring.  I click the switch, the modal disappears immediately and all of a sudden I'm refreshing the manifest.

I would be in favor of waiting to hide the modal for a bit, just so the user can at least see the switch toggle..  maybe run `hideModal()` after a `setTimeout` of 500ms?  @johnpmitsch thoughts?

Reply:
It is a bit jarring, at least the switch indicates an action though. The upload manifest workflow is a bigger concern IMO as it uploads as soon as you select a file.

If we can figure out a way to improve it we can do that, I'm not sure we should set a timeout though. It could be worth asking UX.

I'm really fine with how it is now though, I think its going to be hard to improve in isolation without refactoring all the loading/async workflows.